src/site/content/en/blog/media-session/index.md

```diff
@@ -22,9 +22,10 @@ feedback:
 To let users know what's currently playing in their browser and control it
 without returning to the page that launched it, the Media Session API has been
 introduced. It allows web developers to customize this experience through
-metadata in custom media notifications and media events such as playing,
-pausing, and even seeking and track changing. These customizations are available
-in several contexts including desktop media hubs, media notifications on mobile,
+metadata in custom media notifications, media events such as playing, pausing,
+seeking, track changing, and video conferencing events such as mute/unmute,
```

Comment:
nit: "mute/unmute microphone" to distinguish from muting the audio output

Reply:
Done.